plc4j/tools/connection-cache/src/main/java/org/apache/plc4x/java/utils/cache/LeasedPlcConnection.java

```diff
@@ -176,6 +176,11 @@ public PlcReadRequest.Builder addTagAddress(String name, String tagAddress) {
             public PlcReadRequest.Builder addTag(String name, PlcTag tag) {
                 return innerBuilder.addTag(name, tag);
             }
+
+            @Override
+            public Optional<PlcTag> parseSafe(String tagAddress) {
```

Comment:
Same comment as, I think this would be a better fit in the connection itself, not in the builder.